### ISSUE TITLE
Epicea shows a Change that is not a change

### ISSUE THREAD

**Issue by akevalion** [guillep/pharo-core#3695]
I have used epicea to recover my changes, but I have found an issue.
I have exported this change into a plain text file and zipped it.

Download the attach file and unzip the file. 
Load the file into pharo with the drag and drop interaction.
Then you will see 
![image](https://user-images.githubusercontent.com/10532890/60357808-a2683200-99a2-11e9-83b2-43dca85956c3.png)

This is one change of many changes. I think it is hard to show the entire list, for that reason I only send on change.
Epicea shows this change when there is not change, then Epicea should not show it, or avoid create it

Bug on Pharo8 64 bits, MacOS 10.14.5

[FileOut_2gk.ombu.zip](https://github.com/pharo-project/pharo/files/3340118/FileOut_2gk.ombu.zip)

**Comment by tinchodias** [comment]
I should take a look

**Comment by tinchodias** [comment]
I opened the ombu file in latest P8 and also see an empty diff.  I also opened the file in a text editor and compared the texts of the "old" and "new" instances of `RGClassDefinition`in the change, and don't see a difference. 

@akevalion Thanks for reporting, but the key to understand more is to know HOW to generate this empty change. In other words, if you find steps to reproduce it, then I can work more on it.
